nb2workflow/deploy.py

```diff
@@ -538,6 +613,7 @@ def main():
     parser.add_argument('--local', action="store_true", default=False)
     parser.add_argument('--build-engine', metavar="build_engine", default="docker")
     parser.add_argument('--nb2wversion', metavar="nb2wversion", default=version())
+    parser.add_argument('--ontology_URL', metavar="ontology_URL", default=default_ontology_URL)
```

Comment:
Just to match the style

```suggestion
    parser.add_argument('--ontology-url', metavar="ontology_URL", default=default_ontology_URL)
```

Reply:
Following @dsavchenko suggestion I have changed --ontology_URL to ontology_path. The command line parameter changed to --ontology-path